### DISCUSSION TITLE
UI improvements or exchange

### DISCUSSION THREAD

**Discussion by tebjan** [stride3d/stride#1328] [Contributors]
There are several alternatives to the Stride.UI package. Should we make one of them the new default, or invest energy to improve and fix the current one?

https://github.com/TheKeyblader/Stridelonia
https://github.com/Eideren/StrideCommunity.ImGuiDebug
https://github.com/rds1983/Myra

**Comment by larinius** [comment]
Game UI based on XAML can look very impressive (Noesisengine), but requires a separate Editor (Visual Studio Blend).
If Stridelonia supports themes and same fast to edit in Visual Studio as old WPF it will be great for users. 
We could make inventory grid or main menu literally in minutes using XAML markup language. 

I think ImGui cannot replace "normal GUI" but very convenient for developing tasks, It would be nice to have it.
